lib/iden3comm/domain/use_cases/get_filters_from_iden3msg_use_case.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:polygonid_flutter_sdk/common/domain/use_case.dart';
+import 'package:polygonid_flutter_sdk/credential/domain/use_cases/get_claims_use_case.dart';
+import 'package:polygonid_flutter_sdk/iden3comm/domain/entities/iden3_message_entity.dart';
+import 'package:polygonid_flutter_sdk/iden3comm/domain/entities/jwz_proof_entity.dart';
+import 'package:polygonid_flutter_sdk/iden3comm/domain/entities/proof_request_entity.dart';
+import 'package:polygonid_flutter_sdk/iden3comm/domain/exceptions/iden3comm_exceptions.dart';
+import 'package:polygonid_flutter_sdk/iden3comm/domain/repositories/iden3comm_repository.dart';
+import 'package:polygonid_flutter_sdk/identity/domain/entities/did_entity.dart';
+import 'package:polygonid_flutter_sdk/identity/domain/use_cases/get_did_use_case.dart';
+import 'package:polygonid_flutter_sdk/proof/domain/use_cases/is_proof_circuit_supported_use_case.dart';
+
+import '../../../common/domain/entities/filter_entity.dart';
+import '../../../credential/domain/entities/claim_entity.dart';
+import '../../../credential/domain/use_cases/get_claim_revocation_status_use_case.dart';
+import '../../../credential/domain/use_cases/update_claim_use_case.dart';
+import '../../../identity/domain/repositories/identity_repository.dart';
+import '../../../proof/domain/entities/circuit_data_entity.dart';
+import '../../../proof/domain/repositories/proof_repository.dart';
+import '../../../proof/domain/use_cases/generate_proof_use_case.dart';
+import 'get_proof_requests_use_case.dart';
+
+class GetFiltersFromIden3MsgParam {
+  final Iden3MessageEntity message;
+
+  GetFiltersFromIden3MsgParam({
+    required this.message,
+  });
+}
+
+class GetFiltersFromIden3MsgUseCase
```

Comment:
Same comment about the naming, the specification should be done in Presentation layer.